LOCALIZATION_CONTRIBUTORS

```diff
@@ -80,6 +80,7 @@ John Michael Martinez Tenorio
 Pablo Vicente Greco
 Emanuel Dri (AR)
 Edgar Bruney Castañeda Torres
+Alejandro Medici (AR)
```

Comment:
We do not include the countries where each person is from, please check the list of the other languages, in a later commit the "`(AR)`" of the contributor that also has the country will be removed.
Please remove the parentheses indicating your country.